node/src/types/block.rs

```diff
@@ -486,8 +485,8 @@ impl Display for FinalizedBlock {
             random bit {}, timestamp {}",
             self.era_id,
             self.height,
-            HexList(&self.deploy_hashes),
-            HexList(&self.transfer_hashes),
+            format!("[{}]", itertools::join(&self.deploy_hashes, ", ")),
+            format!("[{}]", itertools::join(&self.transfer_hashes, ", ")),
```

Comment:
```suggestion
            HexList(&self.deploy_hashes),
            HexList(&self.transfer_hashes),
```

Reply:
Fixed in [e93dc37](https://github.com/nilz3ro/casper-node/pull/5/commits/e93dc3787a9983c352ebfdba0fc6bccb8f4d7c41).